eng/scripts/KillProcesses.ps1

```diff
@@ -21,6 +21,10 @@ _kill git.exe
 _kill vctip.exe
 _kill chrome.exe
 _kill h2spec.exe
-iisreset /restart
+_kill WerFault.exe
+if (Get-Command iisreset -ErrorAction ignore) {
```

Comment:
Good idea. Fortunately, the command only writes to `stdout` when process lacks admin rights.